pyeed/fetch/ncbiproteinmapper.py

```diff
@@ -10,7 +12,9 @@
 from pyeed.core.region import Region
 from pyeed.core.dnarecord import DNARecord
 from pyeed.core.annotation import Annotation
-from pyeed.core.proteinrecord import ProteinRecord
+
+if TYPE_CHECKING:
```

Comment:
Not sure how the 'sdrdm' update method will handle this. I guess the way it is currently implemented it will again add all imports defined within the `TYPE_CHECKING` condition to the top of the file.

Reply:
As far as I can recall, it only traverses the `core` module. Or am I wrong? We should definitely leave out other modules than `core`.

Reply:
I think we are okay. Since the `preserve_custom_functions` function is only called on re-rendered files, it should only process content within `core`. Here is a snippet from the code:

```python
def save_rendered_to_file(rendered: str, path: str, use_formatter: bool) -> None:
    """Saves a rendered Object, Enum or Init to a file"""

    if os.path.isfile(path):
        rendered = preserve_custom_functions(rendered, path)

    with open(path, "w") as f:
        f.write(rendered)
```

Reply:
I see, yes it only traverses core. Didn't pay attention...

Reply:
No worries, I was puzzled too 😁